code/game/objects/structures/cargo_container.dm

```diff
@@ -6,10 +6,51 @@
 	bound_width = 32
 	bound_height = 64
 	density = 1
-	var/health = 200
+	var/damage = 0
```

Comment:
This sort of change needs to be done at a lower level like /structure so they can all benefit